README.md

```diff
@@ -17,18 +17,18 @@ To speed up QuiVer Benchmarks you can mount already downloaded text recognition
 ```
 
 to the `volumes` section.
-Otherwise the tool will download all `ocrd-tesserocr-recognize` models as well as `ocrd-calamari-recognize qurator-gt4histocr-1.0` on each run.
+Otherwise, the tool will download all `ocrd-tesserocr-recognize` models as well as `ocrd-calamari-recognize qurator-gt4histocr-1.0` on each run.
 
 ## Usage
 
 - clone this repository
-- [customize](#custom-workflows-and-data) QuiVer Benchmarks according to your needs
-- run `docker compose build && docker compose up`
+- (optional) [customize](#custom-workflows-and-data) QuiVer Benchmarks according to your needs
+- run `docker-compose up --build`
```

Comment:
I should have specified in the prerequisites that I use the Compose plugin. No need for the dash then.

Reply:
Hm I thought the `docker-compose` is more common for not experienced users (like me). Having only one well known command would eliminate possible hurdles.

Reply:
my 2ct :+1: for `docker-compose`

Reply:
`docker-compose` is V1 and `docker compose` is V2. Check [here](https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose). The V1 will/may eventually get dropped at some point, so I think it's better to stick to V2.

Reply:
Ok wow, [here](https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command) official statement linked in some answer there. Maybe we should at least add at point how to install `compose`.

Reply:
Honestly, I did not expect it to be that soon :)

Reply:
> Ok wow, [here](https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command) is an official statement linked in some answer there. Maybe we should at least add a point how to install `compose`.

Sure, I honestly just forgot to put it in the requirements. :)

Reply:
Added in [572e248](https://github.com/OCR-D/quiver-benchmarks/pull/1/commits/572e248a3dc4c89761414be5edeca76b8d0f1ed7)